bindings/python/examples/how_tos/native_tokens/create.py

```diff
@@ -16,16 +16,16 @@
 wallet.set_stronghold_password(os.environ["STRONGHOLD_PASSWORD"])
 
 # Sync account with the node
-response = account.sync()
+balance = account.sync()
 
 # We can first check if we already have an alias in our account, because an alias can have many foundry outputs and therefore we can reuse an existing one
-if len(account.aliases) == 0:
+if len(balance["aliases"]) == 0:
```

Comment:
curiousity: would something like this also work?
```py
if not balance["aliases"]:
```

Reply:
Yes works